ooiservices/config.py

```diff
@@ -73,8 +74,9 @@ def init_app(cls, app):
 
 config = {
     'development': DevelopmentConfig,
+    'local_dev': LocalDevelopmentConfig,
     'testing': TestingConfig,
     'production': ProductionConfig,
 
-    'default': DevelopmentConfig
+    'default': LocalDevelopmentConfig
 }
```

Comment:
configuration should be made via YAML so that administrators whom are not familiar with Python may edit the file. The other reason is so that deployments can be automated for systems.